__tests__/integration/backend-proxy.test.js

```diff
@@ -90,5 +90,33 @@ describe('Backend proxy integration', () => {
 				expect(response.body).toEqual(backendData);
 			});
 	});
+
+	test('proxies a request to redirect', () => {
+		const location = 'http://not.the-back.end/my-nice/location?here=there#my-id';
+		const scope = nock(backend).get('/hello/world')
+			.reply(301, {}, {location});
+
+		return request.get('/usePathOn/hello/world')
+			.then(response => {
+				scope.done();
+
+				expect(response.status).toBe(301);
+				expect(response.headers.location).toEqual(location);
+			});
+	});
+
+	test('proxies a request to redirect and removes backend from the header', () => {
+		const relativePath = `/my-nice/location?here=there#my-id`;
+		const scope = nock(backend).get('/hello/world')
+			.reply(301, {}, {location: `${backend}${relativePath}`});
```

Comment:
Can you use a different `30x` redirect for this test?

Reply:
👍